app/pdf_redactor.py

```diff
@@ -107,6 +107,8 @@ def redactor(options):
 	writer = PdfWriter()
 	writer.trailer = document
 	writer.write(options.output_stream)
+	if message:
+		return message
```

Comment:
you should assign message, or if `options.content_filters` isn't supplied it'll error because message isn't defined.

If you just set it to None then we can always `return message`